Update README.md

haneefa1 opened the pull request:
I updated the biographies section of the press kit with headshot links for the management team.

haneefa1 committed:
Update README.md

I updated the biographies section of the press kit with headshot links for the management team.

contentwillvary commented:
well done for doing PR - please can you put the links in the right hand side per row from 'Headshot' text instead of on the names? otherwise people have to click on a link without knowing what the link is for in this format, and if someone ctrl+f to find headshot they wouldn't find it

haneefa1 committed:
Update README.md

I've moved the headshots for the management team to the links column of the biographies.

haneefa1 commented:
done

haneefa1 committed:
Update README.md

link for your headshot was updated

contentwillvary reviewed (approved):
Great thanks!
